tests/cli.js

```diff
@@ -488,3 +490,11 @@ test('run `cli --quiet` on 1 non-json file', macro.testCLI, {
   args: ['*/package.json', '--quiet'],
   message: 'Should output error message',
 })
+
+test('run `cli --stdin` piping a json file from stdin', async (t) => {
+  const cp = spawn('node', ['cli.js', '--stdin'])
+  cp.stdin.end(`{\n  "description": "Description",\n  "name": "Name"\n}\n\n`)
+
+  const out = await getStream(cp.stdout)
+  t.is(out, `{\n  "name": "Name",\n  "description": "Description"\n}\n\n`)
```

Comment:
If we do `cat package.json | sort-package-json --stdin > package.json`, will there be an extra new line in the file? Maybe we should use `process.stdout.write` instead of `console.log`

Reply:
Yes, it was producing an extra line at the end of the file, and your suggestion of using process.stdout.write worked. Good catch.

For testing manually run:

```sh
cat package.json | sort-package-json --stdin > package2.json && diff package.json package2.json
```

Reply:
Fixed in [`d0e0084` (#296)](https://github.com/keithamus/sort-package-json/pull/296/commits/d0e0084b0196f1c9ea86e72d13af21455215ccd5)